### PR TITLE
CSA-124: Add user roadmap activity 

### PR DESCRIPTION
**Pull request by VovaLyapota**

### CODE REVIEW

File: src/components/ui/Select/Select.tsx [hunk 1]
```diff
@@ -87,31 +85,38 @@ const Select = ({
...
-                  />
-                )}
-              </li>
-            ))}
+            {options?.map((option) => {
+              const optionLabel =
+                typeof option === 'string' ? option : option.label;
+              const optionValue =
+                typeof option === 'string' ? option : option.value;
+
```
Comment (Tanyapavliuk) [inline src/components/ui/Select/Select.tsx]: It is not at all clear what is happening here.

File: src/components/ui/Icon/index.ts [hunk 2]
```diff
@@ -1,2 +1,3 @@
 import Icon from './Icon';
+export * from './Icon';
```
Comment (Tanyapavliuk) [inline src/components/ui/Icon/index.ts:2]: What is the purpose of adding another export?

File: src/components/ui/Select/Select.tsx (hunk 1 above)
Comment (StanislavKharkiv) [inline src/components/ui/Select/Select.tsx]: I think better always pass objects in options. If we don't need label just leave it  the same like value. In this case we always know what type of data we have and pass

File: src/components/RoadmapList/Filters.tsx [hunk 3]
```diff
@@ -43,31 +43,40 @@ const Filters = ({ onSelect, filters, onSearch, onClear }: FiltersProps) => {
         <div className={style.filterBox}>
           <div className={style.filterList}>
             <Select
-              options={categoryOptions}
+              options={categoryOptions.map((option) => ({
+                label: option,
+                value: option,
+              }))}
```
Comment (Tanyapavliuk) [inline src/components/RoadmapList/Filters.tsx:49]: For the future, If you rewrite someone else's code, better to write about it to the author. 

Why duplicate the code twice?

**Review by tymnik — APPROVED** [review]

File: src/components/ui/Select/Select.tsx (hunk 1 above)
Comment (VovaLyapota) [reply src/components/ui/Select/Select.tsx]: It's already been removed

**Review by Tanyapavliuk — APPROVED** [review]

File: src/components/ui/Icon/index.ts (hunk 2 above)
Comment (VovaLyapota) [reply src/components/ui/Icon/index.ts:2]: Make export in Select more clear

**Pull request merged**